RecoMuon/L3MuonProducer/src/L3MuonProducer.cc

```diff
@@ -269,14 +269,16 @@ void L3MuonProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptio
     psd0.add<double>("PCut", 2.5);
     {
       edm::ParameterSetDescription psd1;
-      psd1.add<bool>("DoPredictionsOnly", false);
-      psd1.add<std::string>("Fitter", "hltESPL3MuKFTrajectoryFitter");
-      psd1.add<std::string>("TrackerRecHitBuilder", "hltESPTTRHBWithTrackAngle");
-      psd1.add<std::string>("Smoother", "hltESPKFTrajectorySmootherForMuonTrackLoader");
-      psd1.add<std::string>("MuonRecHitBuilder", "hltESPMuonTransientTrackingRecHitBuilder");
-      psd1.add<std::string>("RefitDirection", "insideOut");
-      psd1.add<bool>("RefitRPCHits", true);
-      psd1.add<std::string>("Propagator", "hltESPSmartPropagatorAny");
+      TrackTransformer::fillPSetDescription(psd1,
```

Comment:
@Martin-Grunewald this update should address your concerns about default values, they are now ensured